packages/runtime-core/__tests__/componentPublicInstance.spec.ts

```diff
@@ -274,14 +274,82 @@ describe('component: proxy', () => {
     const app = createApp(Comp)
 
     app.mount(nodeOps.createElement('div'))
+    
+    // access 'toggle' to ensure key is cached
+    let v1 = instanceProxy.toggle()
+
+    // reconfigure "toggle" to be getter based.
+    let getCalledTimes = 0
+    Object.defineProperty(instanceProxy, 'toggle', {
+      get(){
+        getCalledTimes++
+        return  ()=> 'b'
```

Comment:
iI think you need to format your code: run `pnpm format` to remove the weird indentation.